melos.yaml

```diff
@@ -21,4 +21,4 @@ ide:
 scripts:
   format-check: dart format --output=none --set-exit-if-changed --line-length 120 .
   analyze: dart analyze --fatal-infos .
-  test: melos exec --concurrency=1 --fail-fast --dir-exists=test -- "flutter test"
+  test: melos exec --concurrency=1 --fail-fast --dir-exists=test -- "flutter test --concurrency=$(nproc --all)"
```

Comment:
duplicate concurrency argument

Reply:
But the melos concurrency is for all package and the flutter test concurrency is inside each package. The over all concurrency is limited to 1 to avoid producing garbage output